src/views/Contact.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable arrow-body-style */
+import React from 'react';
+import { connect } from 'react-redux';
+
+class Contact extends React.Component {
```

Comment:
As you are not using the functions yet you can add them later when you need it and write this component in a more essential way:

```suggestion
const Contact = () => (
  <div className="content">
     <h6> Home </h6>
  </div>
);
```
And you make this as functional component, you can later use `useEffect` hook when you need to add some logic. 
This comment applies for Home and Inside pages too.

Reply:
ok I gonna. use the hooks for all the project  :)